modules/SDL.lua

```diff
@@ -195,6 +195,37 @@ function SDL:DeleteFile()
   end
 end
 
+--- Update SDL log4cxx.properties in order SDL will be able to write logs through Telnet
+local function updateSDLLogProperties()
```

Comment:
@dboltovskyi suggest to add 2 more functions:
- `backupSDLLogProperites` for copying `log4cxx.properties` to `backup_log4cxx.properties`
- `restoreSDLLogProperties` for restoring file from `backup_log4cxx.properties` if exists and removing `backup_log4cxx.properties` after restore

Then you could call backup before call update and call restore in the end

Reply:
@AKalinich-Luxoft Yes, it would be great. But this functionality is related to a runner which is actually absent in ATF. Current fix is just a workaround. 
Backup and restore is already implemented in run.sh script